docs/src/modules/toggle.js

```diff
@@ -0,0 +1,34 @@
+function toggle(options = {}) {
```

Comment:
New module for a11y friendly toggle functionality. This is primarily used in the new `CodeBlock` component but can be used more generally as well.